textproc/recoll/Portfile

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# $Id: Portfile 111989 2013-10-08 17:10:55Z ryandesign@macports.org $
```

Comment:
This line is not used anymore since we moved to GitHub, so it should be removed.